src/components/MenuItem.js

```diff
@@ -45,117 +46,122 @@ const defaultProps = {
+                                    ...props.iconStyles,
+                                ]}
+                            >
+                                <Avatar
+                                    imageStyles={[styles.avatarNormal, styles.alignSelfCenter]}
+                                    source={props.icon}
+                                    fallbackIcon={props.fallbackIcon}
+                                />
+                            </View>
+                        )}
+                        <View style={[styles.justifyContentCenter, styles.menuItemTextContainer, styles.flex1]}>
                             <Text
-                                style={styles.textLabelSupporting}
+                                style={titleTextStyle}
+                                numberOfLines={1}
                             >
-                                {props.subtitle}
+                                {props.title}
                             </Text>
+                            {Boolean(props.description) && (
+                                <Text
+                                    style={descriptionTextStyle}
+                                    numberOfLines={2}
+                                >
+                                    {props.description}
+                                </Text>
+                            )}
                         </View>
-                    )}
-                    {Boolean(props.brickRoadIndicator) && (
-                        <View style={[styles.alignItemsCenter, styles.justifyContentCenter]}>
-                            <Icon
-                                src={Expensicons.DotIndicator}
-                                fill={props.brickRoadIndicator === CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR ? colors.red : colors.green}
-                                height={variables.iconSizeSmall}
-                                width={variables.iconSizeSmall}
-                            />
-                        </View>
-                    )}
-                    {props.shouldShowRightIcon && (
-                        <View style={styles.popoverMenuIcon}>
-                            <Icon
-                                src={props.iconRight}
-                                fill={StyleUtils.getIconFillColor(getButtonState(props.focused || hovered, pressed, props.success, props.disabled, props.interactive))}
-                            />
-                        </View>
-                    )}
-                    {props.shouldShowSelectedState && <SelectCircle isChecked={props.isSelected} />}
-                </View>
-            </>
-        )}
-    </Pressable>
-);
+                    </View>
+                    <View style={[styles.flexRow, styles.menuItemTextContainer, styles.pointerEventsNone]}>
+                        {props.badgeText && <Badge text={props.badgeText} badgeStyles={[styles.alignSelfCenter, (props.brickRoadIndicator ? styles.mr2 : undefined)]} />}
```

Comment:
@MariaHCD here is one example. But now when I searched with `badgeText` it looks like it existed before the PR too. I'll find the original PR and comment there too.